pkg/cloud/services/networking/network.go

```diff
@@ -131,8 +131,7 @@ func (s *Service) ReconcileNetwork(openStackCluster *infrav1.OpenStackCluster, c
 		openStackCluster.Status.Network.ID = res.ID
 		openStackCluster.Status.Network.Name = res.Name
 		openStackCluster.Status.Network.Tags = res.Tags
-		sInfo := fmt.Sprintf("Reuse Existing Network %s with id %s", res.Name, res.ID)
-		s.scope.Logger().V(6).Info(sInfo)
+		s.scope.Logger().V(6).Info("Reuse Existing Network", "name", res.Name, "id", res.ID)
```

Comment:
nit: Should probably be 'Reusing'. Probably also ditch the caps after the initial one.